ansible/library/announce_routes.py

```diff
@@ -149,7 +149,8 @@ def change_routes(action, ptf_ip, port, routes):
     wait_for_http(ptf_ip, port, timeout=60)
     url = "http://%s:%d" % (ptf_ip, port)
     data = {"commands": ";".join(messages)}
-    r = requests.post(url, data=data, timeout=90, proxies={"http": None, "https": None})
+    # nosemgrep-next-line
```

Comment:
Can you remove this commented line

Reply:
@judyjoseph since we are using http:// instead of https://, semgrep was failing citing security issue..
This line is required to skip the semgrep error... without this PR checker doesnt pass..